recipe/meta.yaml

```diff
@@ -11,43 +11,49 @@ source:
   url: https://github.com/mysql/mysql-connector-python/archive/{{ version }}.tar.gz
   sha256: {{ sha256 }}
   patches:
-    - 0001-Typecast-password-to-const-char-explicitly-py3.patch
-    - 0002-Cast-uint-to-enum-mysql_enum_shutdown_level-explicit.patch
-    - 0003-Don-t-error-out-if-get_git_info-fails.patch
+    - patches/0001-Typecast-password-to-const-char-explicitly-py3.patch
+    - patches/0002-Fix-location-for-searching-mysql-headers-and-libs-on.patch
+    - patches/0003-Fix-indentation-for-common-flags.patch
+    - patches/0004-Revert-Fix-MacOS-wheels-platform-tag.patch
+    - patches/0005-Replace-google-protobuf-string-with-std-string.patch
+    - patches/0006-Delete-openssl-vendor-ed-libs-only-if-provided.patch
 
 build:
   number: 1
   # VS 2008 is not supported for libprotobuf
   skip: True  # [win and py2k]
```

Comment:
```suggestion
  skip: True  # [win or py<38]
```

Reply:
I want to keep windows build in here, as it was built previously.
Due to discontinuation of `mysql-connector-c`, and unavailability of `mysql` on windows, this build just doesn't support c extension on windows.